provisioning_client/tests/hsm_client_riot_ut/hsm_client_riot_ut.c

```diff
@@ -250,6 +293,32 @@ static int my_DERtoPEM(DERBuilderContext* Context, uint32_t Type, char* PEM, uin
     return 0;
 }
 
+static void my_mbedtls_mpi_free(mbedtls_mpi* X)
+{
+    if (X == NULL)
```

Comment:
We include the `{}` even on one line `if()` statements

Reply:
These ones I needed to mirror exactly how the mbedtls lib handles them, however the brackets shouldn't make any difference in functionality, just readability and congruence. Will add them.

Reply:
Added brackets